calcom/cal.com
# fix: Foreign uppercase characters not recognized as such in signup form

## What does this PR do?

It fixes the bug of not identifying uppercase foreign language letters and identify them an uppercase foreign letter as an uppercase letter rather than registering it as a smallcase.



The changes are made in the validPassword.ts file . 

No dependecies are required for this change to happen.


- Fixes #17779  (GitHub issue number)
- Not Call Number provided in the description as of now



## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?



- Are there environment variables that should be set?

None required

- What are the minimal test data to have?

Use a signup form with input fields for username, email, and password.
For testing uppercase foreign letters, ensure that the test input includes characters with accented letters (e.g., Á, É, Ö, Ç).

- What is expected (happy path) to have (input and output)?

Input: A signup form where the user enters an email or username that contains uppercase foreign letters (e.g., MÁRTIN or CÉLINE).
Output: The form should correctly recognize and accept uppercase foreign letters as uppercase and not throw validation errors, maintaining the case integrity of the entered text.
Ensure the system correctly handles the characters and processes them as uppercase (and not as lowercase or invalid input).

## Checklist



- I haven't checked if my changes generate no new warnings
